src/main/resources/processes/FOI_DRAFT.bpmn

```diff
@@ -154,118 +151,6 @@
     <bpmn:sequenceFlow id="Flow_08u1iug" sourceRef="Gateway_19rmbk9" targetRef="RESPONSE_TYPE">
       <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${ResponseType != "EXEMPTION"}</bpmn:conditionExpression>
     </bpmn:sequenceFlow>
-    <bpmn:sequenceFlow id="Flow_1i0njkg" sourceRef="Gateway_0byag49" targetRef="Gateway_1ma44jq">
-      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${DIRECTION == "FORWARD"}</bpmn:conditionExpression>
-    </bpmn:sequenceFlow>
-    <bpmn:userTask id="SENSITIVITY" name="Sensitivity" camunda:formKey="FOI_SENSITIVITY">
```

Comment:
As with in `hocs-data` can you check with Ed that this definitely isn't required please?

Reply:
There's a thread about this in decs-foi on Slack, started at Jul 29th at 10:06 AM. Ed, asked me to remove it and updated the Mural board to reflect this.

Reply:
Removal agreed